### ISSUE TITLE
replication controller log spam

### ISSUE THREAD

**Issue by eparis** [openshift/origin#13722]
over 7 hours of logs we had over 7,500 log messages like
```
replication_controller.go:322] Observed updated replication controller SNIP. Desired pod count change: 0->0
replication_controller.go:322] Observed updated replication controller SNIP. Desired pod count change: 1->1
```
In order to make online clusters supportable we need to reduce useless logs. The signal/noise ration is quite low.

**Comment by mfojtik** [comment]
@kargakis you probably know better where to look.

**Comment by ncdc** [comment]
That actually comes from https://github.com/kubernetes/kubernetes/commit/a82e54262df1de0ab0a554ff896a0af634b10c95, which was added to debug kubernetes/kubernetes#31981. It was fixed in early September 2016. I'll open a PR to remove it.

**Comment by ncdc** [comment]
https://github.com/kubernetes/kubernetes/pull/44391

**Comment by ncdc** [comment]
Will be fixed when the rebase merges (#13653)
